src/app/modules/bounce/__tests__/bounce.controller.spec.ts

```diff
@@ -1,48 +1,61 @@
+import { ObjectId } from 'bson'
 import { StatusCodes } from 'http-status-codes'
+import mongoose from 'mongoose'
 import expressHandler from 'tests/unit/backend/helpers/jest-express'
 import { mocked } from 'ts-jest/utils'
 
 import { handleSns } from 'src/app/modules/bounce/bounce.controller'
+import getBounceModel from 'src/app/modules/bounce/bounce.model'
 import * as BounceService from 'src/app/modules/bounce/bounce.service'
-import { ISnsNotification } from 'src/types'
+import { IBounceSchema, ISnsNotification } from 'src/types'
+
+const Bounce = getBounceModel(mongoose)
 
 jest.mock('src/app/modules/bounce/bounce.service')
 const MockBounceService = mocked(BounceService, true)
 
+const MOCK_NOTIFICATION = 'someValue'
 const MOCK_REQ = expressHandler.mockRequest({
-  body: ({ someKey: 'someValue' } as unknown) as ISnsNotification,
+  body: ({ Message: MOCK_NOTIFICATION } as unknown) as ISnsNotification,
```

Comment:
nit: Do you actually need to type this?

Reply:
Yeah, otherwise it's not the correct argument type for `handleSns`